### DISCUSSION TITLE
Can I use /_next/image directly?

### DISCUSSION THREAD

**Discussion by RafalFilipek** [vercel/next.js#20090] [Help] [answered]
Hi,

Is it  safe to use `/_next/image` directly without `<Image />` component? Not sure if I can treat it as a part of public API.

Thank you

Rafał

**Comment by zomars** [comment]
This! We we're wondering the same thing the other day. This could be useful to programatically generate images for backgrounds for example.

**Comment by timneutkens** [comment, marked as answer]
It's not part of the public API

**Reply by zomars** [reply]
Is there a reason for that? Or it would too much burden to support?

**Reply by timneutkens** [reply]
Making sure it's future proof, the way optimization works can now be transparently optimized whereas we would not be able to do so if it was a public api.

**Reply by zomars** [reply]
Thanks for the clarification, makes sense!
